MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -133,8 +133,9 @@ public FontDescriptionProcessor()
 
                 output.Texture.Faces[0].Add(face);            
 			}
-			catch(Exception ex) {
-				context.Logger.LogImportantMessage("{0}", ex.ToString());
+			catch(Exception ex)
+			{
+			    throw;

```

Comment:
Rather than simply rethrowing it, this clause doesn't need to be there at all.  The `Exception ex` variable is also not needed and will be issuing a compiler warning about an unused variable.